website/content/api-docs/secret/consul.mdx

```diff
@@ -93,14 +95,13 @@ To create a client token within a particular Consul admin partition:
 
 ### Parameters for Consul versions 1.8 and above
 
-- `consul_namespace` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul namespace the token
-  will be generated within. The namespace must exist, and the policies or roles assigned to the
-  Vault role must also exist inside the given Consul namespace. If not provided, the "default"
-  namespace is used.
+- `consul_namespace` `(string: "")` - Specifies the Consul namespace in which the token is generated within.
+  The namespace must exist, and the Consul policies or roles assigned
+  to the Vault role must also exist inside the given Consul namespace. If not provided, the 
+  namespace `default` is used.
 
-- `node_identities` `(list: <node identity or identities>)` - The list of node identities to
-assign to the generated token. This may be a comma-separated list to attach multiple node identities
-to a token.
+- `node_identities` `(string: "")` - The list of node identities to assign to the generated
```

Comment:
Hm, I'm not sure. In the code the FieldSchema type is `TypeStringSlice` so that it builds the slice when provided multiple times via CLI.  I didn't try passing in a list via CLI to test that, does it work?

Reply:
For the JSON list I think you would do something like

```
$ cat node_identities.json | vault write -f  foo/consul/roles/bar -
```

Reply:
Sounds good, and that also makes more sense.